api/structures.go

```diff
@@ -16,11 +16,19 @@ type MonitoringResponse struct {
 	UpdatedTime time.Time
 }
 
+// Health is a type to indicates health of the unit.
+type Health int
+
+const (
+	unhealthy = 0
```

Comment:
Is there a reason to make these all lower case rather than all CamelCase, like for most other constants? Is it to keep it hidden in the api package?

Reply:
It just feels a bit odd to have package level constants called simply `healthy` and `unhealthy`

Reply:
Youre right. Exported type constants should be also exported.